extensions/amp-ad-exit/0.1/config.js

```diff
@@ -138,3 +148,29 @@ function assertTarget(name, target, config) {
     }
   }
 }
+
+/**
+ * Checks whether a vendor is valid (i.e. listed in vendors.js and has
+ * transport/iframe defined.
+ * @param {string} vendor The vendor name that should be listed in vendors.js
+ */
+function assertVendor(vendor) {
+  user().assert(ANALYTICS_CONFIG &&
+      ANALYTICS_CONFIG[vendor] &&
+      ANALYTICS_CONFIG[vendor]['transport'] !== undefined &&
```

Comment:
do you need the undefined check?  Why not just ANALYTICS_CONFIG[vendor][transport]?  I assume falsey values wouldn't allowed...

Reply:
I believe a previous review comment (earlier PR) said to account for possible falsey values. But as they'll be strings (not numbers/booleans) and couldn't be the empty string, I agree that falsey should not be possible. Changed.